test/raft/randomized_nemesis_test.cc

```diff
@@ -1460,6 +1460,16 @@ class raft_server {
             _fd_subscription = std::nullopt;
             co_await _fd_service->stop();
             co_await _server->abort();
+
+            {
+                std::vector<raft::snapshot_id> snapshot_ids;
+                snapshot_ids.reserve(_snapshots->size());
+                for (const auto& p: *_snapshots) {
+                    snapshot_ids.push_back(p.first);
+                }
+                BOOST_TEST_INFO(format("snapshot ids: [{}]", snapshot_ids));
```

Comment:
I don't think anyone's reading the XML files that this produces. We prefer `seastar::logger`.

Reply:
This message goes into the regular pytest log file along with the messages generated by `seastar::logger`. It is quite useful.